partiql-cli/build.gradle.kts

```diff
@@ -62,6 +61,26 @@ tasks.register<GradleBuild>("install") {
     tasks = listOf("assembleDist", "distZip", "installDist")
 }
 
+tasks.named("build").configure {
+    dependsOn("generateMockDBJAR")
+}
```

Comment:
Also, seems like this causes a bug.  If you run `./gradlew clean`, then run `./gradlew :partiql-cli:test`, there are failures.

This is likely caused by the order of task creation.